src/Microsoft.Framework.Runtime.Interfaces/IAssemblyLoadContext.cs

```diff
@@ -4,10 +4,31 @@
 
 namespace Microsoft.Framework.Runtime
 {
+    /// <summary>
+    /// A context in which assemblies can be loaded.
+    /// </summary>
     public interface IAssemblyLoadContext : IDisposable
     {
+        /// <summary>
+        /// Load an assembly by name.
+        /// </summary>
+        /// <param name="name">The name of the assembly.</param>
+        /// <returns>The loaded assembly.</returns>
```

Comment:
This is something I hate about XML Doc comments. I'd err on the side of over-crefing, but that's personal opinion.